addons/medical_damage/CfgAmmo.hpp

```diff
@@ -0,0 +1,46 @@
+class CfgAmmo {
+    class BulletCore;
+    class BulletBase: BulletCore {
+        ACE_damageType = "bullet";
```

Comment:
Maybe add `ACE_damageType = "bullet";` to `Default
Might cover some odd uses, like `BulletCore`

Reply:
yeah, I forgot about core config stuff